src/Services/Data/DefaultEmoteRegexMatchWriter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace BrackeysBot
+{
+    public class DefaultEmoteRegexMatchWriter
```

Comment:
Is there a way to get around having to generate a code file?

The solution is decent, but having to generate large code files is always a bit ... unsettling.

Reply:
I tried to let the values be generated at runtime when the bot is started, but the operation was really slow O(n²) for around 1900 emote input elements. Another solution might be to have it be ran in a thread and store the data into a variable once the thread has done its work. 

json/yaml files have the issue of requiring the file to be additionally included in the build and then an IO operation. As the default emotes shouldn't be dynamic, I thought it would be fine to include them like that (only when discord adds new ones, it can be added to the bot, but it is not something that every bot instance needs to have as file I feel).